plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -68,6 +68,18 @@ export const DEFAULT_OBJECTS: ObjectToFetch[] = [
     plural: 'horizontalpodautoscalers',
     objectType: 'horizontalpodautoscalers',
   },
+  {
+    group: 'batch',
+    apiVersion: 'v1',
+    plural: 'jobs',
+    objectType: 'jobs',
+  },
+  {
+    group: 'batch',
+    apiVersion: 'v1',
```

Comment:
is this the right version of cronjobs that we use? I thought we were using `v1beta1`?

Reply:
Ahh I see this ties into your other PR and overrides